lib/tasks/post.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+// Include Gulp, plugins and local files
+var argv = require('yargs').argv;
+var fs = require('fs');
+var gulp = require('gulp');
+var mkdirp = require('mkdirp');
+var moment = require('moment');
+
+var utils = require('../utils');
+
+function post() {
```

Comment:
function post() needs to take `title` and  optional `slug` arguments. These need to passed into the function and the handling of `argv.title` below needs to be revisted